byway/src/main/webapp/routepage.js

```diff
@@ -1,175 +0,0 @@
-    if (status == 'OK') {
-      directionsRenderer.setDirections(response);
-    } else {
-      window.alert("Could not calculate route due to: " + status);
-    }
-  });
-}
-
-/** Clear the stops panel in the html */
-function clearStops(){
-  const stopList = document.getElementById('stop-list');
-  if(stopList != null){
-    stopList.innerText = "";
-  }
-}
-
-/** Get the new list of stops from datastore onload */
-function getStopsOnload(){
-  clearStops();
-  fetch('/api/stop')
-  .then(response => response.json())
-  .then((stopsResponse) => {
-    if(stopsResponse != null){
-      stops.push(...stopsResponse);
-    }
-    renderStopsList();
-  });
-}
-
-/** Re-render stop list synchronously */
-function renderStopsList(){
-  clearStops();
-  const stopList = document.getElementById('stop-list');
-  stops.forEach((stop)=>{
-    stopList.appendChild(createStopButton(stop));
-  })
-}
-
-/** Render stop list  
- *  @param {String} stop a String to add as a button in the schedule panel in the html
- *  @return {button} stopBtn a button showing a selected stop
- */
-function createStopButton(stop){
-  const stopBtn = document.createElement('button');
-  stopBtn.innerText = stop;
-  stopBtn.className =  "btn rec-btn";
-  stopBtn.addEventListener("click", function() {
-    // TODO: delete from recs later for better visuals on html
-    stops = stops.filter(function(stopObj){
-      return stopObj != stop;
-    })
-    updateStops();
-  });
-  return stopBtn;
-}
-
-/** Add stop to or delete stop from the stoplist in javascript and in the datastore */
-function updateStops(){
-  renderStopsList();
-  fetch('/api/stop', {method: 'POST', body: JSON.stringify(stops)});
-}
-
-/** Clear the recommendations panel in the html */
-function clearRecs(){
-  const recList = document.getElementById('rec-list');
-  if(recList != null){
-    // clear list
-    recList.innerText = "";
-  }
-}
-
-/** Get the new list of recommendations from servlet onload */
-function getRecsOnload() {
-  clearRecs();
-  fetch('/api/recs')
-  .then(response => response.json())
-  .then((recommendations) => {
-    recs.push(...recommendations);
-    renderRecsList();
-  })
-}
-
-/** Re-render recs list synchronously */
-function renderRecsList(){
-  clearRecs();
-  const recsList = document.getElementById('rec-list');
-  recs.forEach((rec)=>{
-     recsList.appendChild(createRecButton(rec));
-  })
-}
-
-/** Render stop list  
- *  @param {String} rec a String to add as a button in the recommendations panel in the html
- *  @return {button} recBtn a button showing a recommended place
- */
-function createRecButton(rec){
-  const recBtn = document.createElement('button');
-  recBtn.innerText = rec;
-  recBtn.className =  "btn rec-btn";
-  recBtn.addEventListener("click", function() {
-    // TODO: add to recs later for better visuals on html
-    stops.push(rec);
-    updateStops();
-  });
-  return recBtn;
- }
-
-/* exported initMap */
-/* global google */
-
```

Comment:
Do you want this file to be deleted from master? If not you should commit back the version of the file in master, by fetching it with `git checkout master byway/src/main/webapp/routepage.js` and then committing that.

Reply:
Will add back, and delete the Recommendation.java file that I don't need anymore but is in master.